docs/admins.rst

```diff
@@ -367,6 +367,31 @@ Example:
 .. _quay: https://quay.io/application/
 .. _`config.json`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/config.json
 
+Including OpenShift build annotations in Koji task output
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is possible to include a ``build_annotations.json`` file in the task output
+of successful container image builds. This file may include any wanted
+OpenShift build annotations for the container build triggered by the Koji task
+in question.
+
+The ``koji-containerbuild`` plugin looks for a
+``koji_task_annotations_whitelist`` annotation in the OpenShift build
+annotations. This key should hold a list of annotations to be whitelisted for
+inclusion in the ``build_annotations.json`` file.
+
+If an empty ``build_annotations.json`` file would be generated through the
```

Comment:
If ``koji_task_annotations_whitelist`` is empty, ``build_annotations.json`` file is ommited from the task output

Reply:
This is not the only possible cause for the file to be omitted though...
For instance, the whitelist could be populated with annotations not present in the OpenShift build annotations. 

I added a sentence with possible causes for the file being omitted.

Reply:
ok